galaxy_ng/app/tasks/index_registry.py

```diff
@@ -0,0 +1,160 @@
+import json
+import logging
+from urllib.parse import quote, urlencode
+
+from django.core import exceptions
+from django.utils.translation import gettext_lazy as _
+from django.http.request import HttpRequest
+
+from pulpcore.plugin.tasking import dispatch
+
+from pulp_container.app import models as container_models
+
+from galaxy_ng.app.api.ui import serializers
+from galaxy_ng.app import models
+
+
+log = logging.getLogger(__name__)
+
+
+CATALOG_API = "https://catalog.redhat.com/api/containers/v1/repositories"
+
+
+class CouldNotCreateContainerError(Exception):
+    def __init__(self, remote_name, error=""):
+        self.message = _(
+            f"Failed to create container {remote_name}. {error}"
+        )
+
+        super().__init__(self.message)
+
+
+def _get_request(request_data):
+    request = HttpRequest()
+
+    for k in request_data:
+        setattr(request, k, request_data[k])
+
+    return request
+
+
+def _parse_catalog_repositories(response_data):
+    containers = []
+
+    for container in response_data['data']:
+        containers.append({
+            "name": container['repository'],
+            "upstream_name": container['repository'],
+            "description": container['display_data']['short_description'],
+            "readme": container['display_data']['long_description_markdown'],
+        })
+
+    return containers
+
+
+def _update_distro_readme_and_description(distro, container_data):
+    readme = models.ContainerDistroReadme.objects.get_or_create(container=distro)[0]
```

Comment:
```suggestion
    readme, _ = models.ContainerDistroReadme.objects.get_or_create(container=distro)
```

Reply:
Won't this overwrite `from django.utils.translation import gettext_lazy as _`?

Reply:
You are right, so

`readme, _created = ...`